src/tasks/netcore/NetCoreTaskHelper.ts

```diff
@@ -287,6 +296,49 @@ export class NetCoreTaskHelper implements TaskHelper {
 
         return volumes;
     }
+
+    private async buildWithDotnetSdk(context: DockerBuildTaskContext, buildDefinition: DockerBuildTaskDefinition): Promise<void> {
+        //reference: https://learn.microsoft.com/en-us/dotnet/core/docker/publish-as-container

```

Comment:
Consider adding this comment as a JSDoc @see entry on the method (https://jsdoc.app/tags-see.html)

Reply:
Nice! will do